.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: 22.6.0
     hooks:
       - id: black
-        language_version: python3.9
+        language_version: python3.10
```

Comment:
You can revert this when merged, I don't have python 3.9. If you insist I can download.